src/vss_tools/vspec/vssexporters/vss2csv.py

```diff
@@ -36,53 +36,43 @@ def print_csv_header(file, uuid, entry_type: AnyStr, include_instance_column: bo
         "Allowed",
         "Default",
     ]
-    if uuid:
-        arg_list.append("Id")
-    if include_instance_column:
-        arg_list.append("Instances")
-    file.write(format_csv_line(arg_list))
-
-
-# Format a data or header line according to the CSV standard (IETF RFC 4180)
-
-
-def format_csv_line(csv_fields):
-    formatted_csv_line = ""
-    for csv_field in csv_fields:
-        if csv_field is None:
-            csv_field = ""
-        formatted_csv_line = (
-            formatted_csv_line + '"' + str(csv_field).replace('"', '""') + '",'
```

Comment:
I notice some changes in output of CSV exporters. This line was added once upon the time to handle at least some common scenarios, like if a description/comment contain a comma or a double quote, and then the string has to be quoted according to https://datatracker.ietf.org/doc/html/rfc4180

Now it seems we skip any form of escaping, or?

Reply:
I think the csv writer library handles escaping for us. And yes, the output of all csv's changed and removed unnecessary quoting

Reply:
But I will check some corner cases I guess

Reply:
Would be quite easy to add some test cases unless we already have some.

Reply:
It works fine. I honestly see no real reason to test the official csv lib